GitUI/CommandsDialogs/FormEditor.cs

```diff
@@ -161,7 +171,15 @@ protected override bool ProcessCmdKey(ref Message msg, Keys keyData)
                     Close();
                     return true;
                 case Keys.Control | Keys.S:
-                    SaveChanges();
+                    try
+                    {
+                        SaveChanges();
+                    }
+                    catch (Exception ex)
+                    {
+                        MessageBox.Show(this, $"{_cannotSaveFile.Text}{Environment.NewLine}{ex.Message}", Strings.Error, MessageBoxButtons.OK, MessageBoxIcon.Error);
+                    }
```

Comment:
Do not copy & paste MessageBoxes. Factor out this to a new private function, e.g. `SaveChangesShowException`.
Do you have a better name, @RussKie?